components/event-bus/internal/knative/subscription/controller/subscription/reconcile.go

```diff
@@ -152,7 +152,15 @@ func (r *reconciler) reconcile(ctx context.Context, subscription *eventingv1alph
 		if err != nil && !errors.IsNotFound(err) {
 			return false, err
 		} else if errors.IsNotFound(err) {
-			knativeChannel, err := r.knativeLib.CreateChannel(knativeChannelProvisioner, knativeChannelName, knativeSubsNamespace, timeout)
+
+			//Adding the event-metadata as channel labels
+			knativeChannelLabels := make(map[string]string)
+			knativeChannelLabels["kyma-source-id"] = subscription.SourceID
```

Comment:
Could you please add the "kyma-* strings as const like the other "consts"?

Reply:
OK